src/HIE/Bios/Ghc/Check.hs

```diff
@@ -53,22 +54,23 @@ checkSyntax cradle files = withGhcT $ do
 check :: (GhcMonad m)
       => [FilePath]  -- ^ The target files.
       -> m (Either String String)
-check fileNames = withLogger setAllWarningFlags $ setTargetFiles (map dup fileNames)
+check fileNames = do
+  topDir <- G.topDir <$> G.getDynFlags
+  withLogger (setAllWarningFlags topDir) $ setTargetFiles (map dup fileNames)
 
 dup :: a -> (a, a)
 dup x = (x, x)
 
 ----------------------------------------------------------------
 
 -- | Set 'DynFlags' equivalent to "-Wall".
-setAllWarningFlags :: DynFlags -> DynFlags
-setAllWarningFlags df = df { warningFlags = allWarningFlags }
+setAllWarningFlags :: FilePath -> DynFlags -> DynFlags
+setAllWarningFlags topDir df = df { warningFlags = allWarningFlags topDir }
```

Comment:
this should rather "libdir"?

Reply:
The GHC api seems to refer to the library directory as topDir, that's the function I'm using to get it, but I'm happy out to keep it as libDir for consistency. The true bikeshed question though is, is it libDir or libdir? 🤔